tables/vpc_flow_log/vpc_flow_log.go

```diff
@@ -40,12 +40,11 @@ type VpcFlowLog struct {
 	Region                  *string    `json:"region,omitempty"`
 	SrcAddr                 *string    `json:"src_addr,omitempty"`
```

Comment:
Missing `RejectReason` (Version 8 field) see https://docs.aws.amazon.com/vpc/latest/userguide/flow-log-records.html#flow-logs-fields

Reply:
You have a description for the column in GetColumnDescriptions() also